client/my-sites/plugins/plugin-information/index.jsx

```diff
@@ -62,16 +61,14 @@ export default React.createClass( {
 			return;
 		}
 		return (
-			<div>
-				<a
-					className="plugin-information__external-link"
-					target="_blank"
-					onClick={ analytics.ga.recordEvent.bind( analytics, 'Plugins', 'Clicked wp.org Plugin Link', 'Plugin Name', this.props.plugin.slug ) }
-					href={ 'https://' + this._WPORG_PLUGINS_URL + this.props.plugin.slug + '/' }
-				>
-					<Gridicon size={ 12 } icon="external" />{ this.translate( 'WordPress.org Plugin page' ) }
-				</a>
-			</div>
+			<ExternalLink
```

Comment:
it's so good to see all that html structures get replaced by shared components... :)